.gitignore

```diff
@@ -1,3 +1,5 @@
 stanford-corenlp-full-2018-10-05/
 __pycache__
 config.py
+env
+.vscode/
```

Comment:
I see an unidentified character here.

Reply:
- "env" is my virtualenv.
- .vscode/ must've been generated while I was working in the editor.
Should I remove these? I don't think they can cause an issue for the other collaborators because these will be ignored.

Reply:
@vishakha-lall - Are you talking about the EOL character?
@Mansi145 - As a general practice, add an empty line at the end of the file.

Reply:
@athityakumar yes

Reply:
@vishakha-lall So is there something that I can do about this?
@athityakumar Thanks for letting me know. I'll make sure I add an empty line at the end of the file from the next time.